ui/lib/feed/src/components/editable-reflection/index.tsx

```diff
@@ -69,6 +69,8 @@ const EditableReflection: React.FC<ReflectCardProps & { reflectToId: string }> =
     onCompleted: async () => {
       setEdit(false);
       setNewContent(null);
+      //reset the state modified inside Editor component when done
+      setIsReflecting(true);
```

Comment:
Is this meant to be `false` if you are resetting ?

Reply:
No it means `true ´ because we need to reset the state that would be set as `false` inside the child component

Reply:
To make the intent clear what if you create another component inside the same folder something like ReflectionEditorRenderer or something simmilar that wraps the reflection editor along with the state to show and hide the editor. And also rename the local state to showEditor rather than `isReflecting`. That way you don't need to reset it here and the intent will be more clear.

Reply:
You created this component a while ago Didd :)). I will try your suggestion to make it clearer

Reply:
Yes but the initial intent has been changed ever since :) Thanks.

Reply:
Yes. But this the component has evolved over time and what I saw on this PR made it clear the naming and the reset here doesn't make sense :) Thanks.